acceptance/setup/aio/010_Install.rb

```diff
@@ -31,10 +31,7 @@ def install_from_local(hosts, package)
       :puppet_agent_sha     => ENV['SHA'],
       :puppet_agent_version => ENV['SUITE_VERSION'] || ENV['SHA']
     }
-    agents.each do |agent|
-      next if agent == master # Avoid SERVER-528
-      install_puppet_agent_dev_repo_on(agent, opts)
-    end
+    install_from_build_data_url('puppet-agent', "http://builds.delivery.puppetlabs.net/puppet-agent/#{opts[:puppet_agent_sha]}/artifacts/#{opts[:puppet_agent_sha]}.yaml")
```

Comment:
This removes the need for the `opts` object completely. Just use `ENV['SHA']` directly.

Reply:
What this does preclude is anyone outside Puppet running these tests.

Reply:
Also, this applies to all hosts. That means this setup needs to be slightly restructured.